src/woo_publications/publications/api/serializers.py

```diff
@@ -340,6 +346,20 @@ def update(self, instance, validated_data):
         return publication
 
 
+class PublicationUpdateSerializer(PublicationSerializer):
+    class Meta(PublicationSerializer.Meta):
+        fields = PublicationSerializer.Meta.fields
+
+        extra_kwargs = {
+            "bron_bewaartermijn": {
+                "required": False,
+            },
+            "archiefactiedatum": {
+                "required": False,
+            },
+        }
```

Comment:
I think we have problems with these fields being writable - shouldn't these be calculated/derived in the publicatiebank?